patches/server/0001-Concrete-POM-Changes.patch

```diff
@@ -103,9 +103,9 @@ index 674096cab190d62622f9947853b056f57d43a2a5..ac94d08c50a86d0339d4b6c16b3b3f1f
 @@ -11,7 +11,7 @@ public final class Versioning {
      public static String getBukkitVersion() {
          String result = "Unknown-Version";
- 
+
 -        InputStream stream = Bukkit.class.getClassLoader().getResourceAsStream("META-INF/maven/com.destroystokyo.paper/paper-api/pom.properties");
 +        InputStream stream = Bukkit.class.getClassLoader().getResourceAsStream("META-INF/maven/ca.spottedleaf.concrete/concrete-api/pom.properties"); /* Concrete */
```

Comment:
and this

Reply:
This is intentional, this would require changing class names and group ids which is not something in the scope of the PR nor something I'm sure leaf would want anyone but themselves handling

Reply:
Yep, you're right. my bad!